docs/scheduling.md

```diff
@@ -5,63 +5,75 @@ title: Scheduling
 Scheduling guidelines
 -------------
 
-### Access-tier based volume scheduling
+### Volume scheduling based on drive labels
 
-In addition to scheduling based on resource constraints (available space) and node topology (affinity/anti-affinity etc.), it is possible to further influence the scheduling of workloads to specific volumes based on "access-tiers". DirectPV pre-defines 3 access tiers:
+In addition to scheduling based on resource constraints (available space) and node topology (affinity/anti-affinity etc.), it is possible to further influence the scheduling of workloads to specific volumes based on drive lavels. The DirectPV drives can be labeled based on its classsification and such labels can be used in the storage class parameters to control scheduling to pick up the chosen drives for volumes. 
 
-- Hot
-- Warm
-- Cold
+By default, the DirectPV drives will not have any user defined labels set on them. Use `kubectl directpv label drives` command to set user defined labels to DirectPV drives.
 
-By default, directpv drives are not associated with any access-tier. An admin can associate drives to access tiers. Further instructions on the configuration is provided in the following sections.
+**Notes:**
 
-#### Step 1: Set access-tier tag on the drives
+- This applies only for creating new volumes as this is a schedule time process.
```

Comment:
So if you apply a label to an existing volume, that would only affect new pods scheduled that could use that volume, not existing volumes which were already scheduled/provisioned.

Reply:
Scheduling is controlled here by drive labels. Labels can be set on the drive and used in storage class parameters to control scheduling of "new" volumes. So, that the workloads using the new storage class will only get the volumes created on the drives with the respective labels.

So, these drive labels are only used at the time of scheduling new volumes. Existing volumes which are already scheduled doesn't mind these labels.